changelogs/fragments/163-documentation-broken-links.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - documentation - documentation urls are fixed for digital_ocean_domain_record and digital_ocean_droplet_info modules. (https://github.com/ansible-collections/community.digitalocean/pull/163)
```

Comment:
You can drop the scope here, it can look like this:

```yaml
bugfixes:
  - Documentation URLs are fixed for the C(digital_ocean_domain_record) and C(digital_ocean_droplet_info) modules (https://github.com/ansible-collections/community.digitalocean/pull/163).
```
Note the capitalization and period after the reference URL.

Reply:
Fixed it. Thanks for the guidance.